UPGRADING.md

```diff
@@ -1,3 +1,87 @@
+Upgrading from v0.67.1 to v0.71.4
+=================================
+
+Read through for a list of the major changes between versions 0.67.1 and 0.71.4, covering:
+* [Configuration changes](#configuration-changes)
+* [Command line changes](#command-line-changes)
+
+# Configuration changes
+
+This document covers most of the breaking changes in using the command line or configuring the different softwares. 
+
+We also recommend that node operator generates a bare configuration (`vega init...`, `vega datanode init...`) in a temporary folder to compare this new configuration to their existing ones to address any changes. Any missing entries in the configuration will be using defaults.
+
+## Vega
+
+The Vega configuration file can be found in `$VEGA_HOME/config/node/config.toml`.
+
+## Data node
+
+The data node configuration file can be found in `$DATANODE_HOME/config/data-node/config.toml`.
+
+### Settings added in 0.71.4
+
+**_Rate Limiting_** - In the past the datanode API did rate limiting only by the number of subscriptions open per IPs. With the new version a rate limit per IP on the unary calls for the RESY, GraphQL and grpc APIs was introduced. The server now allows 20 API calls per second on average with a burst of up to 100 api calls in a short period of time.
+For details how to use this setting and configure this functionality, please refer to [rateLimiting](https://github.com/vegaprotocol/vega/blob/develop/datanode/ratelimit/README.md)
+
+**_REST and GraphQL configuration_** - Previously the configuration of the Graphql and REST APIs used separate HTTP servers and configuration. This was merge onto a single server now. Also the default endpoint for Graphql changed from /query to /graphql
+Usage example:
+```Toml
+[Gateway]
+  Port = 3008
+  IP = "0.0.0.0"
+  [Gateway.GraphQL]
+    Enabled = true
+    Endpoint = "/graphql"
+  [Gateway.REST]
+    Enabled = true
+```
+
+**_IPFS update_** - The ipfs library used by the datanode to share network history have been updated. This may require manual steps to reset the state and apply the ipfs migrations.
```

Comment:
```suggestion
**_IPFS update_** - The IPFS library used by the data node to share network history has been updated. This may require manual steps to reset the state and apply the IPFS migrations.
```